aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModule.java

```diff
@@ -1717,6 +1741,26 @@ public void conclude()
                     cycleThresholdNs);
             }
 
+            if (null == snapshotDurationTracker)
+            {
+                snapshotDurationTracker = new SnapshotDurationTracker(
+                    ClusterCounters.allocate(
+                        aeron,
+                        buffer,
+                        "Consensus module max snapshot duration in ns",
```

Comment:
The naming is misleading as it does not measure the `ConsensusModule` snapshot time but instead measures the total snapshot time.

Reply:
So both the names and static final constants should reflect that. Also the system property for the entire thing should just use `aeron.cluster.` prefix and the ones for CM would then use `aeron.cluster.consensus.module.` prefix.